Program.cs

```diff
@@ -104,11 +101,12 @@ static void Main(string[] args)
             }
             finally
             {
-                socket.Close();
+                foreach (var socket in Sockets)
+                    socket?.Close();
             }
         }
 
-        class CommandLineArguments
+        private class CommandLineArguments

```

Comment:
this line is not changing behaviour, so let's not add private keyword here, we can do that in a refactoring PR

Reply:
Ah, I have several extensions like Resharper, CodeMaid ... that fix and clean up codes